ui/src/app/cohort-review/detail-tabs/detail-tabs.component.html

```diff
@@ -1,6 +1,6 @@
 <clr-tabs>
   <clr-tab>
-    <button clrTabLink id="detail-summary-tab"(click) = "getDomainsParticipantsData()">Summary</button>
+    <button clrTabLink id="detail-summary-tab">Summary</button>
```

Comment:
when we are switching between other tabs summary tab clicking is not smooth it takes time to open the tab. Not sure what is causing the issue.

Reply:
I noticed this as well on certain participants. I think there are some issues with the data manipulation in the charts, and it ends up doing substantial extra work when there are many data points, and this leads to a noticeable pause on render. I think this is already happening without this PR, but it's masked because we reload the data every time, so the initial render just shows the spinner. We should investigate that separately.